new/PRS14018Astaraniqos.xml

```diff
@@ -0,0 +1,60 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14018Astaraniqos" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Aristarchus</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </t:langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-05-10">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">Aristarchus</persName>
+                    <persName xml:lang="gez" xml:id="n2">ʾAstarāniqos</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">አስተራኒቆስ፡</persName>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14018Astaraniqos" passive="LIT1295Dersan#HomilySane"/>
+                    <relation name="betmas:husbandOf" active="PRS14018Astaraniqos" passive="PRS00000Euphemia"/>
+                    <relation name="betmas:locusTarget" active="#PRS14018Astaraniqos" passive="LOC2177Cilici"/>
```

Comment:
remove `#` (actually activity in Cilicia can be put to `floruit`)

Reply:
I removed the hash-sign. How can I introduce a LOC-ID to the floruit encounter?
I did not find the word for Cilicia (Qəlqəyā) in the text, but it was mentioned by Marrassini. I don't know for sure, whether Astaraniqos and Afomyo worked or originated in Cilicia - if this makes a difference.

Reply:
`<floruit>The hagiographic account places him in <placeName ref="LOC2177Cilici"/></floruit>`

Reply:
Done for Astaraniqos and for Afomya.